deploy/resources/v0.3.1/release.yaml

```diff
@@ -424,20 +424,20 @@ spec:
         - -stderrthreshold
         - INFO
         - -kubeconfig-writer-image
-        - gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter@sha256:a5a2a193f01c2dd47da7306560de3eb52db7b88a587a1a53a34e450c8e12bb57
+        - quay.io/openshift-pipeline/tektoncd-pipeline-kubeconfigwriter:v0.3.1
```

Comment:
just curious if you manually munged the release.yaml?

Reply:
In short, No
I used a downstream release.yaml which vincentD sent me
@vdemeester 

But it was my first approach as downstream release.yaml was not available

Reply:
it would be nice if downstream pipeline repo publishes the `release.yaml` just like upstream as a downloadable artifact.

Reply:
Absolutely :+1: 
i have created an issue in jira
https://jira.coreos.com/browse/SRVKP-178